### DISCUSSION TITLE
application.yaml profiles stop working in Quarkus version 2.10.0.Final

### DISCUSSION THREAD

**Discussion by InigoCrespoSoria** [quarkusio/quarkus#26329] [Q&A] [answered]
I have configured `io.quarkus:quarkus-config-yaml` in my application.

But, I can't use `"%staging"` type profile definitions inside application.yaml

Every time I run .`/gradlew check -Dquarkus.test.profile=test` all the defined profiles inside application.yaml configuration are considered and not only the `"%test"` profile.

This was not happening with the previous version 2.9.2.Final

Is this a known issue?

**Comment by radcortez** [comment, marked as answer]
We are not aware of any issues with profiles and YAML. Are you able to provide us with a reproducer? Thank you!
